docker-compose.yml

```diff
@@ -140,8 +140,8 @@ services:
       context: .
       dockerfile: ./packages/content-fetch/Dockerfile-local
```

Comment:
@jacksonh why are we using a `Dockerfile-local` here? Is it something different from the `Dockerfile`?

Reply:
@sywhb i think there was some issue running Dockerfile locally with docker-compose vs Dockerfile which was running in App Engine. 

Probably an issue with the Chromium dependency

Reply:
I didn't have such an issue. Maybe it is good to remove the `Dockerfile-local` now?

Reply:
@sywhb yeah probably ok now. Maybe need to do a full test with --force-recreate also

Reply:
Yeah, I tested